pkg/notary/authenticating_round_tripper.go

```diff
@@ -39,28 +41,34 @@ func (a *AuthenticatingRoundTripper) RoundTrip(req *http.Request) (*http.Respons
 
 	header := resp.Header.Get("www-authenticate")
 	parts := strings.SplitN(header, " ", 2)
```

Comment:
should `len(parts) == 2` be checked?

Reply:
I'm not checking it here because I assume the response is well formed as `Bearer options...`. That seems like a fair assumption to me, but I totally hear you.